js/pinpay/pinpay_hosted.js

```diff
@@ -42,6 +42,10 @@ var PinpayHosted = Class.create({
   },
 
   receiveMessage: function(e) {
+    // Make sure we're only getting messages from PIN related sources.
+    if (e.origin == "https://cdn.pin.net.au") {
```

Comment:
@moloughlin Yeah, when I originally write it it was looking specifially for the one source I wanted to filter not "everything that's not Pin".  Fixed now.